cli/cmd/stat.go

```diff
@@ -362,17 +359,17 @@ func printSingleStatTable(stats map[string]*row, resourceTypeLabel, resourceType
 	for _, key := range sortedKeys {
 		namespace, name := namespaceName(resourceTypeLabel, key)
 		values := make([]interface{}, 0)
-		templateString := "%s\t%s\t%.2f%%\t%.1frps\t%dms\t%dms\t%dms\t%.f%%\t%d\t\n"
+		templateString := "%s\t%s\t%.2f%%\t%.1frps\t%dms\t%dms\t%dms\t%d\t\n"
 		templateStringEmpty := "%s\t%s\t-\t-\t-\t-\t-\t-\t-\t\n"
```

Comment:
If you're removing the TLS from `templateString`, you also need to remove it from `templateStringEmpty`, otherwise you get stray dashes: 

![Screen Shot 2019-04-02 at 9 51 16 AM](https://user-images.githubusercontent.com/2289389/55421263-b6300380-552d-11e9-8210-62ef1a1d444e.png)

To fix, you'd change this line:
`templateStringEmpty := "%s\t%s\t-\t-\t-\t-\t-\t-\t-\t\n"`
to 
`templateStringEmpty := "%s\t%s\t-\t-\t-\t-\t-\t-\t\n"`

and do the same on line 367.